src/emcpy/stats.py

```diff
@@ -285,3 +285,28 @@ def bootstrap(insample, level=.95, estimator='mean', nrepl=10000):
     ci_upper = _np.percentile(deltas, upper_pctile)
 
     return ci_lower, ci_upper
+
+
+def calc_bins(data, eval_type='omf'):
+    """
+    Calculate number of bins and binsize for histograms based on data size.
+
+    Args:
+        data: (array like) histogram data
+        eval_type: (str, default 'omf') options are 'omf', 'oma', 'observation', 'hofx'
+
+    Returns:
+        bins: (int) number of bins
+        binsize: (float) size of bins
+    """
+    binsize = (_np.max(data) - _np.min(data))/_np.sqrt(len(data))
+
+    if eval_type.lower() == 'omf' or eval_type.lower() == 'oma':
+        start = 0 - (4 * _np.std(data))
```

Comment:
Do you think we should make the number of standard deviations an option? We can set the default to `nstdevs=4`. @CoryMartin-NOAA what do you think?

Reply:
Yeah probably should be an option. 4 standard deviations covers a lot/most of the distribution but I bet 3 is probably good enough for many applications.

Reply:
Ok, I'll make that change.  Give me an hour or so; I'm in the middle of something else now.